src/code-challenges/.map/array-filter-sort/array-sort.js

```diff
@@ -0,0 +1,39 @@
+export function leastToGreatest(arr) {
+    return arr.sort((a, b) => {
+        if (a > b) return 1;
+        if (a < b) return -1;
+        return 0;
+    })
+}
+
+export function greatestToLeast(arr) {
+    return arr.sort((a, b) => {
+        if (a > b) return -1;
+        if (a < b) return 1;
+        return 0;
+    })
+}
+
+export function lengthSort(arr) {
+    return arr.sort((a, b) => {
+        if (a.length < b.length) return -1;
+        if (a.length > b.length) return 1;
+        return 0;
+    })
+}
+
+export function alphabetical(arr) {
+    return arr.sort((a, b) => {
```

Comment:
totally works! 
There is something called a localeCompare() that does the same thing. 
```
return arr.sort((a, b) => a.localeCompare(b));
```